packages/core/content-manager/server/tests/index.test.e2e.js

```diff
@@ -91,15 +91,11 @@ describe('Relations', () => {
       expect(body.id).toBeDefined();
       expect(body.name).toBe('tag1');
       expect(body.createdBy).toMatchObject({
-        firstname: 'admin',
```

Comment:
No, the firstname was incorrect. I believe it's because the test order changed and the firstname/lastname were changed in a previous test. So I just removed them as I don't think it's relevant to test that here.